### DISCUSSION TITLE
Have Bookmark Panel Remain Open Permanently (No auto-hide)

### DISCUSSION THREAD

**Discussion by kenreeser** [mbnuqw/sidebery#1412] [Ideas]
I used to use Tab Tree and another add-on by the same author that would allow tabs to be viewed at the top of the sidebar and the bookmarks to viewed approximately half way down the sidebar (and you could customize the height of the bookmarks subpanel). The important capability was the bookmarks panel would remain open (no auto-hiding). I would love to have this added as a preference. Thanks very much and keep up the awesome development.

**Comment by noraef** [comment]
I very much agree with that suggestion. I like the bookmark subpanel a lot more than the separate, top bookmark panel (which I've deleted), because I can see my tabs at the same time as I see my bookmarks. But if subpanels could be persistent, without auto-hiding, it would be even better!
